Buildings/Experimental/DHC/Examples/Combined/SeriesVariableFlowAgentControl.mo

```diff
@@ -0,0 +1,266 @@
+within Buildings.Experimental.DHC.Examples.Combined;
+model SeriesVariableFlowAgentControl
```

Comment:
The Example icon looks like the wrong resolution in the package manager.  This should be fixed.

Reply:
Properly resized icon